packages/perseus/src/perseus-types.ts

```diff
@@ -909,6 +909,79 @@ export type PerseusGraphTypeRay = {
     startCoords?: CollinearTuple;
 } & PerseusGraphTypeCommon;
 
+type AngleGraphCorrect = {
+    type: "angle";
+    // Whether to show the angle measurements.  default: false
```

Comment:
Kept this comment so it's easy to tell what the default is.

Reply:
I think this comment might be meant for a different field? The line below this is `allowReflexAngles?: boolean;`, which controls whether angles of more than 180° are allowed.

Reply:
I think the default boolean is false. When the validator is scoring angle graphs, it sends the the value for `allowReflexAngles` to `getClockwiseAngle`. In that function, the default value for `allowReflexAngles` is false. So if the rubric does not set a value for it, it automatically goes to false. It seems like it would never be undefined though. Would you agree with that? Might be just a holdover from `PerseusGraphTypeAngle`
<img width="431" alt="image" src="https://github.com/user-attachments/assets/05d71fc1-ffce-4831-ba55-30f11411fea8">

Reply:
Removed the comment as I made the field required for the rubric.